libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/route/NavigationMapRoute.java

```diff
@@ -445,7 +445,7 @@ private void addDirectionWaypoints() {
   }
 
   private void addUpcomingManeuverArrow(RouteProgress routeProgress) {
-    if (routeProgress.upcomingStepPoints() == null || routeProgress.upcomingStepPoints().size() < TWO_MANEUVERS) {
+    if (routeProgress.upcomingStepPoints() == null || routeProgress.upcomingStepPoints().size() < TWO_MANEUVERS || routeProgress.currentStepPoints().size() < TWO_MANEUVERS) {
```

Comment:
Our checkstyle runs a 120 character limit on each line, can we break this into two lines?  Or better, can we extract each `||` into individual `boolean` conditions?  For example 
```
boolean invalidUpcomingStepPoints = routeProgress.upcomingStepPoints() == null 
    || routeProgress.upcomingStepPoints().size() < TWO_MANEUVERS;
```

Reply:
You are totally right, sorry I didn't think about a character limit.